BigANNBinaryEmbeddingScearioBase.cs

```diff
@@ -132,6 +120,14 @@ protected async Task PerformIngestion(IngestionOperationType ingestionOperationT
             await Task.WhenAll(tasks);
         }
 
+        // check if scenario needs to read label files 
+        protected bool IsWithLabelScenario()

```

Comment:
let us not leak info of "amazon" in base class. This is a shortcut that is adding bad debt. Can we use something similar like what we did for streaming (we don't have any method like IsWithStreamingScenario())? This maybe verbose in short term but will really help in long term as this app continues to get more sophisticated.